mergify_engine/worker.py

```diff
@@ -1201,11 +1231,12 @@ async def async_status() -> None:
     worker_count: int = worker_per_process * process_count
 
     redis_stream = utils.create_yaaredis_for_stream()
-    org_bucket_selector = SharedOrgBucketSelector(redis_stream, 0, worker_count)
 
     def sorter(item: typing.Tuple[bytes, float]) -> int:
         org_bucket, score = item
-        return org_bucket_selector.get_shared_worker_id_for(org_bucket)
+        return SharedOrgBucketSelector.get_shared_worker_id_for(
+            org_bucket, worker_count
```

Comment:
I'm confused, what is worker_count here? where is it defined?

Reply:
L1231

Reply:
Ok I missed this was a closure 🤦